### PR TITLE
[develop] Bindings - pass runtime table parameter to proof creation

### PR DESCRIPTION
**Pull request by dannywillems**
Explain your changes:
* This PR, with their counterparts in https://github.com/o1-labs/snarkyjs/pull/1031 and https://github.com/o1-labs/snarkyjs-bindings/pull/85 add the parameter "runtime_tables" to the proof creation. For the moment, an empty array is given.
* See PR train : https://github.com/MinaProtocol/mina/issues/13086#issuecomment-1630930227

It will be followed by a PR adding runtime tables to the proof inputs. A WIP is already available in [this branch](https://github.com/MinaProtocol/mina/commits/dannywillems/track-runtime-table-cfg) and [here](https://github.com/o1-labs/snarky/pull/828). Please do not have a look now.

Explain how you tested your changes:
* locally running `./run-ci-tests` and relying on the CI in snarkyjs. The current expectation is to have the current tests running without any error. No semantic is added.

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

**Comment by dannywillems** [comment]
!ci-build-me

**Comment by dannywillems** [comment]
!ci-build-me

**Comment by dannywillems** [comment]
!ci-build-me

File: src/lib/crypto/kimchi_bindings/stubs/src/main.rs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 use kimchi::circuits::{
     expr::FeatureFlag,
+    lookup::runtime_tables::caml::{CamlRuntimeTable, CamlRuntimeTableCfg},
```
Comment (mrmr1993) [inline src/lib/crypto/kimchi_bindings/stubs/src/main.rs:3]: Nit: it feels like this didn't need to move out of the group

File: src/lib/pickles/pickles.ml [hunk 2]
```diff
@@ -1733,9 +1733,11 @@ module Make_str (_ : Wire_types.Concrete) = struct
                             ~f:(fun { Impls.Wrap.Proof_inputs.auxiliary_inputs
                                     ; public_inputs
                                     } () ->
+                              (* TODO(dw) pas runtime tables information *)
```
Comment (mrmr1993) [inline src/lib/pickles/pickles.ml:1736]: En Anglais, on dise 'not'. Ou 'pass' si une erreur :)

**Review by mrmr1993 — APPROVED** [review]

File: src/lib/pickles/pickles.ml (hunk 2 above)
Comment (dannywillems) [reply src/lib/pickles/pickles.ml]: It was a typo :). Fixed in 9339d908b0c81b84175357449f9af2744cc3ed82.

**Comment by dannywillems** [comment]
Rerunning the CI as I push-forced after fixing up commits and regenerated the bindings.

**Comment by dannywillems** [comment]
!ci-build-me

**Pull request merged**